samples/unit-tests/maps/setdata/demo.js

```diff
@@ -867,4 +870,13 @@ QUnit.test('Map set data with updated data (#3894)', function (assert) {
         before,
         'The view should not change after updating data values'
     );
+
+    const mapNavY = chart.mapNavButtons[0].alignAttr.y,
+        exportIconY = chart.exportSVGElements[0].alignAttr.translateY;
+
+    assert.notEqual(
+        mapNavY,
+        exportIconY,
+        '#15782, mapNav should not overlap with export icon.'
+    );
```

Comment:
Also, I'm wondering about how safe this test is. What about the buttons overlap, but there's only 1px difference in y? This test would not catch this.
What about testing something like this?
```
assert.ok(
    mapNavY > exportIconY + exportIconY's height,
    '#15782, mapNav should not overlap with export icon.'
);
```